gooey/python_bindings/gooey_decorator.py

```diff
@@ -53,6 +52,7 @@ def Gooey(f=None,
 
   def build(payload):
     def run_gooey(self, args=None, namespace=None):
+      from gooey.gui import application

```

Comment:
Could you add a quick one-liner comment explaining why the import is deferred (so I don't blindly move it back out at some later date)